### ISSUE TITLE
ParamsErrors don't highlight input fields for deep params

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#518]
If an error in a CbrainTask's params is recorded, where the param is deep inside the hash, as in the *paramspath* `myhash[mykey]` or deeper:

```ruby
task.params_errors["myhash[mykey]"] = "is bad"
```

then in the task's param's page the field will not by highlighted in red. The reason is that the ID of the input field will be provided by the `.to_la_id()` method and will be shown as

```html
<input id="cbrain_task_params_myhash_mykey">
```

whereas in the params_errors object they key is stored as the symbol `:"cbrain_task_params_myhash[mykey]"` (yes a symbol with brackets in its name).

This was convenient because it allowed the entire API of ParamsErrors to always provide the developer a consistent interface using *paramspaths*; however now Rails fail to match the ID and the fact that an error was recorded.

Not sure how to solve that yet; maybe stuff the errors object with a duplicate key `:cbrain_task_params_myhash_mykey` key ?

**Comment by prioux** [comment]
Got a solution working using a new to_la_id() (and reverse) in an experimental branch...

**Comment by prioux** [comment]
Fixed by #520
